exam001/Views/ViewController.swift

```diff
@@ -0,0 +1,482 @@
+//
+//  ViewController.swift
+//  exam001
+//
+//  Created by Adnan Al-Akhrass on 11/8/18.
+//  Copyright © 2018 Adnan Al-Akhrass. All rights reserved.
+//
+
+//
+
+import UIKit
+import Firebase
+
+enum textfieldIndex: Int {
+    case first
+    case second
+    case third
+    case fourth
+}
+
+struct Requestes {
+    var fullName: String
+    var floor: String
+    var location: String
+    var items: String
+}
+
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, FireBaseModelDelegate {
+
+    // MARK: - Properties
+    private var firstTextField: CustomTextField!
```

Comment:
UI properties should be lazy initialized 
    ```
lazy var firstTextField: CustomTextField = {
          let textField = CustomTextField()
          CustomTextField(fieldPlaceHolder: viewModel.fieldsArray[textfieldIndex.first.rawValue], fieldColor: .orange, fieldBorderStyle: .line)
          return textField
    }()
```

Reply:
do i use lazy in the setup function for each element or in the properties mark?